src/_pytest/main.py

```diff
@@ -19,12 +20,15 @@
 from _pytest.runner import collect_one_node
 
 # exitcodes for the command line
-EXIT_OK = 0
-EXIT_TESTSFAILED = 1
-EXIT_INTERRUPTED = 2
-EXIT_INTERNALERROR = 3
-EXIT_USAGEERROR = 4
-EXIT_NOTESTSCOLLECTED = 5
+
+
+class ExitCode(enum.IntEnum):
```

Comment:
Please add some docs here, including some sphinx directives so the members show up in `reference.rst` automatically?